### PR TITLE
adds "use strict" clause and use of const and let wherever possible

### PR DESCRIPTION
**Pull request by nunofaria11**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -1,23 +1,24 @@
-var express = require('express');
-var path = require('path');
-var favicon = require('serve-favicon');
-var logger = require('morgan');
-var cookieParser = require('cookie-parser');
-var bodyParser = require('body-parser');
-var ReCAPTCHA = require('recaptcha2');
+"use strict";
```
Comment (butlerx) [inline app.js]: single quotes please

Comment (butlerx) [inline app.js]: single quotes please

File: bin/www [hunk 2]
```diff
@@ -1,25 +1,25 @@
 #!/usr/bin/env node
-
+"use strict";
```
Comment (butlerx) [inline bin/www]: single quotes

File: app.js (hunk 1 above)
Comment (nunofaria11) [inline app.js]: sorry didn't notice the coding style, done

Comment (butlerx) [inline app.js]: no problem

**Pull request merged**